rust/relay/src/main.rs

```diff
@@ -36,44 +36,43 @@ struct Args {
     /// The address of the local interface where we should serve our health-check endpoint.
     ///
     /// The actual health-check endpoint will be at `http://<health_check_addr>/healthz`.
-    #[arg(long, env, default_value = "0.0.0.0:8080")]
+    #[arg(long, env, hide = true, default_value = "0.0.0.0:8080")]
     health_check_addr: SocketAddr,
     // See https://www.rfc-editor.org/rfc/rfc8656.html#name-allocations
     /// The lowest port used for TURN allocations.
-    #[arg(long, env, default_value = "49152")]
+    #[arg(long, env, hide = true, default_value = "49152")]
     lowest_port: u16,
     /// The highest port used for TURN allocations.
-    #[arg(long, env, default_value = "65535")]
+    #[arg(long, env, hide = true, default_value = "65535")]
     highest_port: u16,
-    /// Firezone admin portal websocket URL
-    #[arg(long, env, default_value = "wss://api.firezone.dev")]
-    portal_url: Url,
+    #[arg(long, env, hide = true, default_value = "wss://api.firezone.dev")]
+    api_url: Url,
     /// Token generated by the portal to authorize websocket connection.
     ///
     /// If omitted, we won't connect to the portal on startup.
-    #[arg(long, env)]
-    portal_token: Option<SecretString>,
+    #[arg(env)]
```

Comment:
cc @thomaseizinger -- `long` was removed here to deter users from passing the secret as a CLI arg, just a headsup.